src/PrestaShopBundle/Form/Admin/Sell/Product/Description/RelatedProductType.php

```diff
@@ -53,4 +54,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
         ;
     }
+
+    public function configureOptions(OptionsResolver $resolver)
```

Comment:
Seems like it's not really useful now

Reply:
I removed whole type eventually, just added custom block prefix in entry_options as fields are the same so the form theme still works as expected and we have less code :thinking: